benchmarks/README.md

```diff
@@ -5,7 +5,30 @@ This directory contains benchmarking schemas and data sets that you
 can use to test a variety of performance scenarios against your
 protobuf language runtime.
 
-The schema for the datasets is described in `benchmarks.proto`.
+## Benchmark tools and build instructions
```

Comment:
I think "Prerequisite" is a more appropriate section title here. It should cover:
1. Download dataset.
2. Install cmake (for C++), mvn (for Java)
3. Install protocol compiler
4. Install protobuf C++ runtime (for C++)

Reply:
Done